### PR TITLE
disable flatten for new service

### PR DESCRIPTION
**Pull request by ChenxiJiang333**
https://github.com/Azure/sdk-release-request/issues/4871
https://github.com/Azure/sdk-release-request/issues/4808
https://github.com/Azure/sdk-release-request/issues/4828
https://github.com/Azure/sdk-release-request/issues/4838
https://github.com/Azure/sdk-release-request/issues/4836
https://github.com/Azure/sdk-release-request/issues/4878
https://github.com/Azure/sdk-release-request/issues/4844

### CODE REVIEW

**Review by msyyc — APPROVED** [review]

**Comment by msyyc** [comment]
This PR is only for Python SDK release, so please don't block it by Go SDK generation failure.

**Comment by raych1** [comment]
The Go SDK generation failure are caused by the swagger issue of `ApiCenter` service. But `ApiCenter` is asked to convert spec into TypeSpec.  We need to wait them to make TypeSpec ready instead of fixing the swagger directly.

@lirenhe @ArthurMa1978 @weshaggard please help force merge this PR.
//CC: @msyyc

**Review by raych1 — APPROVED** [review]

File: specification/workloads/resource-manager/readme.python.md [hunk 1]
```diff
@@ -17,3 +17,8 @@ clear-output-folder: true
 no-namespace-folders: true
 output-folder: $(python-sdks-folder)/workloads/azure-mgmt-workloads/azure/mgmt/workloads
 ```
+
+``` yaml $(python)
+modelerfour:
+  flatten-models: false
+```
```
Comment (msyyc) [inline specification/workloads/resource-manager/readme.python.md:24]: https://github.com/Azure/sdk-release-request/issues/4844 is not for workloads which is already GAed. I think it shall be https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.python.md

Comment (ChenxiJiang333) [reply specification/workloads/resource-manager/readme.python.md:19]: updated

**Comment by msyyc** [comment]
move to https://github.com/Azure/azure-rest-api-specs/pull/27380

**Pull request closed without merging**